examples/benchmark/tls_bench.c

```diff
@@ -904,6 +963,14 @@ static int bench_tls_client(info_t* info)
             goto exit;
         }
 
+        if (info->group != 0) {
+            ret = wolfSSL_UseKeyShare(cli_ssl, info->group);
+            if (ret != WOLFSSL_SUCCESS) {
+                fprintf(stderr, "error setting client key share.\n");
```

Comment:
Should consider logging and skipping group, not erroring out.

Reply:
We error out because during SetupSupportedGroups() we checked to make sure any groups that we benchmark  are in fact supported so this is a bad thing if suddenly we get an error.